From akpm@linux-foundation.org  Fri Apr 27 11:05:06 2007
From: "S.Caglar Onur" <caglar@pardus.org.tr>
Date: Thu, 26 Apr 2007 00:38:03 -0700
Subject: USB: Remove duplicate define of OHCI_QUIRK_ZFMICRO
To: greg@kroah.com
Cc: linux-usb-devel@lists.sourceforge.net, akpm@linux-foundation.org, caglar@pardus.org.tr, tony.olech@elandigitalsystems.com
Message-ID: <200704260738.l3Q7c3RZ024376@shell0.pdx.osdl.net>


From: "S.Caglar Onur" <caglar@pardus.org.tr>

Remove duplicate define of OHCI_QUIRK_ZFMICRO from ftdi-elan.c, its already
defined in drivers/ush/host/ohci.c

Signed-off-by: "S.Caglar Onur" <caglar@pardus.org.tr>
Cc: <tony.olech@elandigitalsystems.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/ftdi-elan.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/misc/ftdi-elan.c
+++ b/drivers/usb/misc/ftdi-elan.c
@@ -2304,7 +2304,6 @@ static int ftdi_elan_checkingPCI(struct 
 #define OHCI_QUIRK_SUPERIO 0x02
 #define OHCI_QUIRK_INITRESET 0x04
 #define OHCI_BIG_ENDIAN 0x08
-#define OHCI_QUIRK_ZFMICRO 0x10
 #define OHCI_CONTROL_INIT OHCI_CTRL_CBSR
 #define OHCI_INTR_INIT (OHCI_INTR_MIE | OHCI_INTR_UE | OHCI_INTR_RD | \
         OHCI_INTR_WDH)
